.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rust-analyzer.runnables.extraEnv": {
+        "MOMENTO_API_KEY": "your_api_key_here"
+    }
+}
```

Comment:
Hmm. I already have this environment variable in my profile, so it's already set. This would clobber that. Also, `settings.json` usually has more IDE/user-specific settings (overrides from user/workspace VS Code settings). Can you rename this file to `.vscode/settings.json.default` or `.template` and the CONTRIBUTING?